app/src/pages/api/business/schema.graphql

```diff
@@ -8,8 +8,31 @@ type Business {
 type BusinessCampaign {
   btcPayServerStoreId: String
   businessId: String
+  content: BusinessCampaignContent
+  expiresAt: Date
   id: String!
+  investmentMultiple: Float
   slug: String
+  totalSatsInvested: Float
+}
+
+type BusinessCampaignContent {
+  asHtmlString: String
+  category: String
+  country: String
+  description: String
+  establishedAt: String
+  instagram: String
+  latitude: String
+  longitude: String
+  markerIcon: String
```

Comment:
As we discussed, let's remove this and render it by a `new Map()` of `category => icon.name`